cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -315,12 +315,15 @@ class CloudInfraCatalogItem(BaseCatalogItem):
 
     @property
     def fill_dict(self):
+        # In 5.10 catalog name is appended with 'My Company'
+        catalog_name = getattr(self.catalog, 'name', None)
+        cat_name = catalog_name if self.appliance.version < '5.9' else 'My Company/' + catalog_name
```

Comment:
Required:
Please go with `string` formatting style like
`"My Company/{}".format(catalog_name)`

As per inline comment form, 5.10 its comes with `My Company` right? so condition should `<5.10` what you think?

Reply:
yup that was a typo .